talkingscoresapp/views.py

```diff
@@ -85,9 +103,13 @@ def score(request, id, filename):
     return HttpResponse(template.render(context, request))
 
 # View for midi files to serve with CORS header
+# use GET query string to generate the correct midi file
 def midi(request, id, filename):
-    fr = FileResponse(open("staticfiles/data/" + id + "/" + filename, "rb"))
+    mh = MidiHandler(request.GET, id, filename)
+    midiname = mh.get_or_make_midi_file()
+    fr = FileResponse(open(BASE_DIR+"/staticfiles/data/" + id + "/" + midiname, "rb"))
```

Comment:
Consider using os.path.join?

Reply:
I think it was using os.path.join originally.  I'll have another look at that section.

Reply:
Fixed in 76e0b40